.github/workflows/validate-brewfiles.yml

```
@@ -0,0 +1,39 @@
+name: Validate Brewfiles
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    paths:
+      - "brew/**"
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
+
+      - name: Set up Homebrew
+        uses: Homebrew/actions/setup-homebrew@main

```

```suggestion
        uses: Homebrew/actions/setup-homebrew@8c478e7f5c2c2df9efba834a3568d4fe35cdb2a3

```
I think we should make codacy happy, they don't seem to be updating it super often so the spam is fine in my opinion